src/xcode/ENA/ENA/Source/Client/HTTP Client/WifiOnlyHTTPClient.swift

```diff
@@ -60,6 +61,12 @@ final class WifiOnlyHTTPClient: ClientWifiOnly {
 			}
 		}
 
+		#if !RELEASE
+		guard !disableHourlyDownload else {
+			return
```

Comment:
Don't we have to call the completion with the same error as if we are connected over cellular and `allowsCellularAccess` is set to `false`?

Reply:
exactly, the defer paragraph farther up makes that

Reply:
Hm but `responseError` never gets set and then no completion is called?